### PR TITLE
Draft: update ibc denom tut

### PR DESCRIPTION
**Pull request by tmsdkeys**
Documentation content update for [Tutorials, Understanding IBC denoms]

This PR updates the IBC denom tutorial.

Content:
- Intro to IBC (brief + reference)
- Intro to ICS20 (brief + reference)
- example using mintscan showing `ibc/...` in the wild
- derivation of ibc denom
- denomtrace with `gaiad`
- identify via client, not chain id
- chain registry/CNS

**Notes for reviewer(s)**: Intent of the tutorial is to provide an explanation of what IBC denoms are for users/devs who do not necessarily visit the developer portal doc on IBC. Meaning it **should be able to act as standalone resource** (inviting the reader optionally to do further research on IBC). Please bear this in mind when reviewing and indicate in the comments if you feel it fails this goal!

The changes in this Pull Request include (please tick all that apply):

* [ ] Small language/grammar fixes
* [X] Small content fixes 
* [X] Addition of new content
* [X] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### R1: Internal review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed
- Technical platform review (remove if not required)
  - [ ] requested
  - [ ] completed

### R2: External review

- Technical review
  - [x] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R3: Internal QA review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [x] requested
  - [x] completed

### R4: Final external QA review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [x] requested
  - [ ] completed

### RC: Release Candidate

- [ ] Ready to be merged

### CODE REVIEW

**Review by adiraviraj — CHANGES_REQUESTED** [review]

File: tutorials/understanding-ibc-denoms/index.md [hunk 1]
```diff
@@ -5,201 +5,378 @@ order: 0
...
-## Find the Path of Another Blockchain
+Using this data as source, one could write an API that allows to query for the path, base denom without querying a node.
+
+A relayer can create a new `channel` from a newly created (without an established identity) to another blockchain without revealing too much of its information. Storing path information in the IBC denom that we can trace back and checking the associated client from the channel allows us to make an estimation of the security guarantees of the asset.
+
+### Ensure the IBC Client Isn't Expired
+
+Next to verifying the identity of the chain (or rather light client), another thing to consider is whether the light client is expired.
+
+In the event that Tendermint consensus fails (if >1/3 of validators produce a conflicting block, this is called a _double signing_), _and_ proof of this consensus failure is submitted on-chain, the IBC client becomes frozen with a `frozen_height` that is nonzero. In the previous example, the output of `gaiad query ibc channel client-state` confirms the client status and you know the IBC client is not expired.
```
[the remark this answers is not in the document]
Comment (tmsdkeys) [reply tutorials/understanding-ibc-denoms/index.md:348]: Hmm, not sure if this is the right place to elaborate on more types of misbehaviour. But I can change the sentence to make it clear double signing is just one type of misbehaviour that can lead to expired clients

File: tutorials/understanding-ibc-denoms/index.md [hunk 2]
```diff
@@ -5,201 +5,378 @@ order: 0
...
+Imagine two blockchains, blockchain A and blockchain B. As a starting point, you have some tokens on blockchain A you want to send to blockchain B. You can follow the steps in the image below:
 
 ![IBC token transfer](/tutorials/understanding-ibc-denoms/ibc_token.png "IBC token transfer")
-*Sending token from blockchain A to blockchain B*
+_Sending tokens from blockchain A to blockchain B_
 
-The value that tokens represent can be transferred across chains, but the token itself cannot. When sending the tokens with IBC to another blockchain:
+When sending the tokens with IBC to another blockchain:
 
-1. Blockchain A locks the tokens and relays proof to blockchain B.
```
[the remark this answers is not in the document]
Comment (tmsdkeys) [reply tutorials/understanding-ibc-denoms/index.md:23]: Ah, turns out I already did ;)

File: tutorials/understanding-ibc-denoms/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adiraviraj) [reply tutorials/understanding-ibc-denoms/index.md:348]: Yeah that makes sense

File: tutorials/understanding-ibc-denoms/index.md [hunk 3]
```diff
@@ -5,201 +5,378 @@ order: 0
...
+</HighlightBox>
+
+## Practical example: `denom_trace`
+
+We can distinguish two cases of interacting with IBC denoms:
+
+1. to calculate the IBC denom for a given path
+2. to trace the path information and base denom when encountering an IBC denom
+
+The `transfer` IBC module exposes queries for both of these cases. In order to query, you'll have to interact with a node of a chain.
```
Comment (AndrewB9lab) [inline tutorials/understanding-ibc-denoms/index.md:167]: @tmsdkeys  _"...you'll have to interact with **a node of a chain**."_

Is this use of "node" synonymous with _"a node of a _**network**_"_?

Comment (tmsdkeys) [reply tutorials/understanding-ibc-denoms/index.md:167]: Yes, if network is more consistent with the rest of the content that's fine to change

File: tutorials/understanding-ibc-denoms/index.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
 ---
 parent:
-title: Understand IBC Denoms with Gaia
+title: Understanding IBC denoms with Gaia
```
Comment (AndrewB9lab) [inline tutorials/understanding-ibc-denoms/index.md:3]: I note that the in page title doesn't include "with Gaia". I didn't want to presume which alternative was preferred, so I've left that element of the title unchanged.

Comment (tmsdkeys) [reply tutorials/understanding-ibc-denoms/index.md:3]: Right, forgot about that one. I think we should update and leave off 'with Gaia' as the steps followed are easily extensible to other chains/binaries!

**Comment by tmsdkeys** [comment]
@CitMC all good post QA review (just the image has disappeared in the preview but I guess that's due to new file structuring and will be okay when merged?)

**Review by adiraviraj — APPROVED** [review]

**Review by coldice — APPROVED** [review]

**Pull request merged**